core_backend/app/auth/config.py

```diff
@@ -1,10 +1,12 @@
 import os
 
-CONTENT_READONLY_PASSWORD = os.environ.get("CONTENT_READONLY_PASSWORD", "readonly")
+USER1_USERNAME = os.environ.get("USER1_USERNAME", "user1")
```

Comment:
Why do we need this anymore? I assume this is now in the Db right?

Reply:
Yeah this is in the DB, because of the `core_backend/add_users_to_db.py` script. It didn't occur to me to update this to fetch from the db instead of the env var since it's the same thing right now. 

I'm going to have to read the retrieval key from the DB in PR #179 anyway so will also read the username from there then. We can decide if we want to also store passwords in the DB at that point (I don't think so though - see point about offloading user management to another service like [Fief](https://docs.fief.dev/) above.)